src/test/java/com/uber/cadence/workflow/MetricsTest.java

```diff
@@ -83,5 +110,13 @@ public void testWorkflowMetrics() throws InterruptedException {
     verify(reporter, times(1)).reportCounter("test-done", null, 1);
     verify(reporter, times(1)).reportCounter("test-child-started", null, 1);
     verify(reporter, times(1)).reportCounter("test-child-done", null, 1);
+
```

Comment:
Why do you need that sleep above? 
In any case never use Thread.sleep in workflow unit tests. Use env.sleep instead.

Reply:
This is for the metrics client to flush the cached values out. It's not related to workflow testing.